test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml

```
@@ -299,3 +299,42 @@ tests:
     alert_rule_test:
       - eval_time: 10m
         alertname: ChainsControllerDeadlock
+
+
+  # ----- PAC Controller Potential Deadlock ----
+  - interval: 1m
+    input_series:
+
+      # see https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/ for explanations of the expanding notation used for the values
+      - series: 'pac_watcher_work_queue_depth{source_cluster="cluster01"}'
+        values: '300+300x780'
+      - series: 'pac_watcher_work_queue_depth{source_cluster="cluster01"}'
```

```suggestion
      - series: 'pac_watcher_work_queue_depth{source_cluster="cluster02"}'
```

You missed a nuance @divyansh42 with the labeling between

https://github.com/redhat-appstudio/o11y/blob/beee5d1ad74a1db6de2250a5e32096f38ee7e052/test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml#L271

and 

https://github.com/redhat-appstudio/o11y/blob/beee5d1ad74a1db6de2250a5e32096f38ee7e052/test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml#L269

See if you can figure it out during the earlier part of your day on Monday, and then we'll sync up per usual on Monday morning for me / later for you